perllib/FixMyStreet/App/Controller/Auth.pm

```diff
@@ -47,6 +47,7 @@ sub general : Path : Args(0) {
     if (!$data_username && !$data_password && !$data_email) {
         $c->detach('social/facebook_sign_in') if $c->get_param('facebook_sign_in');
         $c->detach('social/twitter_sign_in') if $c->get_param('twitter_sign_in');
+        $c->detach('social/oidc_sign_in') if $c->get_param('oidc_sign_in');
```

Comment:
Should we rename this `sign_in` with three different values - looks like that would help simplify this in a few places in this commit? Or is there a reason they're separate.

Reply:
Do you mean the function name or the parameter?

Reply:
I think I meant parameter here.
But I think I mentioned function elsewhere, though probably more work/less worth it.